### PR TITLE
Initial prototype

### PR DESCRIPTION
**Pull request by EpyonProjects**
Uses GM4's Custom Crafters 1.17- destroy and 1.18+ storage crafting style

### CODE REVIEW

**Review by iGalaxyYT — CHANGES_REQUESTED** [review]
The advancements of modern technology allowing for non-italic names! (Not the same as the original version but looks better imo)
edit: GH desktop giving me issues and not including the changes :/

**Review by iGalaxyYT — CHANGES_REQUESTED** [review]
for real this time!

File: power_juices/mcpeachpies_power_juices/data/mcpeachpies_power_juices/loot_tables/recipes/apple_juice.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "rolls": 1,
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:potion",
+          "functions": [
+            {
+              "function": "minecraft:set_name",
+              "entity": "this",
+              "name": "Apple Juice"
```
Comment (iGalaxyYT) [inline power_juices/mcpeachpies_power_juices/data/mcpeachpies_power_juices/loot_tables/recipes/apple_juice.json:23]: This should be replaced with `{ "text": "Apple Juice", "italic": false }` (and the same for others) to get rid of the italics in the name (not the same as the original Power Juices but imo looks better)

Comment (iGalaxyYT) [reply power_juices/mcpeachpies_power_juices/data/mcpeachpies_power_juices/loot_tables/recipes/apple_juice.json:23]: yeah this comment will have to be good enough, can't figure out how requesting changes works 🥴

Comment (EpyonProjects) [reply power_juices/mcpeachpies_power_juices/data/mcpeachpies_power_juices/loot_tables/recipes/apple_juice.json:23]: If that's desired, sure.  I can change it tomorrow

Comment (iGalaxyYT) [reply power_juices/mcpeachpies_power_juices/data/mcpeachpies_power_juices/loot_tables/recipes/apple_juice.json:23]: I'll just merge this PR and do it myself actually, it's a simple change, no need to agonize over the process I realize

**Pull request merged**